index.js

```diff
@@ -0,0 +1,214 @@
+        key.innerHTML += `<p class="centerLetter">${item.engValue}</p>`;
+      } else key.innerHTML = `<p class="centerLetter">${item.shiftEngValue}</p>`;
+    } else if (item.view === 'double') {
+      key.innerHTML = `<p class="upLetter">${item.shiftRuValue}</p>`;
+      key.innerHTML += `<p class="centerLetter">${item.ruValue}</p>`;
+    } else key.innerHTML = `<p class="centerLetter">${item.shiftRuValue}</p>`;
+    document.querySelector('.keyboard').append(key);
+  });
+}
+
+
+function createBase() {
+  const main = document.createElement('main');
+  document.querySelector('body').prepend(main);
+  const input = document.createElement('textarea');
+  document.querySelector('main').prepend(input);
+  const keyboard = document.createElement('section');
+  keyboard.className = 'keyboard';
+  document.querySelector('main').append(keyboard);
+}
+
+createBase();
+document.querySelector('textarea').focus();
+document.querySelector('textarea').setAttribute('placeholder', 'Keyboard designed for WINDOWS\nTo change the language click right shift and right alt!\nEnjoy ^__^');
+document.querySelector('body').setAttribute('onselectstart', 'return false');
+
+
+if (languageOfKeyboard === 'eng') {
+  keyFill('eng');
+} else {
+  keyFill('ru');
+}
+
+
+// START OF EVENTS
+
+document.querySelector('body').addEventListener('mousedown', (e) => {
+  cursor = document.querySelector('textarea').selectionStart;
+  const parent = e.target.parentElement;
+  const elem = e.target;
+  if (elem.tagName === 'DIV') elem.classList.add('buttonClick');
+  else if (parent.tagName === 'DIV') parent.classList.add('buttonClick');
+  objOfKeys.forEach((item) => {
+    if (elem.classList.contains(item.code) || parent.classList.contains(item.code)) {
+      if (elem.classList.contains('ShiftLeft') || parent.classList.contains('ShiftLeft') || elem.classList.contains('ShiftRight') || parent.classList.contains('ShiftRight')) pressShift = true;
+      enterSymbols(item);
+      window.event.preventDefault();
+    }
+  });
+});
+
+
+document.querySelector('body').addEventListener('mouseup', (e) => {
+  const parent = e.target.parentElement;
+  const elem = e.target;
+  if (parent.classList.contains('buttonClick') || elem.classList.contains('buttonClick')) {
+    if (elem.classList.contains('CapsLock') || parent.classList.contains('CapsLock')) {
+      if (pressCaps) {
+        if (elem.tagName === 'DIV') elem.classList.remove('buttonClick');
+        else if (parent.tagName === 'DIV') parent.classList.remove('buttonClick');
+      }
+      pressCaps = (!pressCaps);
+    } else {
+      if (elem.tagName === 'DIV') elem.classList.remove('buttonClick');
+      else if (parent.tagName === 'DIV') parent.classList.remove('buttonClick');
+      if (elem.classList.contains('ShiftLeft') || parent.classList.contains('ShiftLeft') || elem.classList.contains('ShiftRight') || parent.classList.contains('ShiftRight')) pressShift = false;
```

Comment:
You can create a separate function for this IF statement as it is very long and uses twice. For example:
function toggleShift(elem, parent, newShiftValue) {
  if (isEnyoneContains([elem, parent], ['ShiftLeft', 'ShiftRight'])) {
      pressShift = newShiftValue;
}
}